GovUkHtmlHelperExtensions.cs

```diff
@@ -254,7 +254,8 @@ public static class GovUkHtmlHelperExtensions
             Expression<Func<TModel, TEnum?>> propertyLambdaExpression,
             FieldsetViewModel fieldsetOptions = null,
             HintViewModel hintOptions = null,
-            string classes = null)
+            string classes = null,
+            Dictionary<TEnum, string> labelDictionary = null)

```

Comment:
Actually, should this be `Dictionary<TEnum, LabelViewModel>` ?

Reply:
In similar vein to what we've done for the others, probably, will fix that